ansible_collections/arista/avd/examples/dual-dc-l3ls/intended/structured_configs/dc1-leaf1a.yml

```diff
@@ -8,143 +8,147 @@ router_bgp:
   - graceful-restart
   - maximum-paths 4 ecmp 4
   peer_groups:
-    MLAG-IPv4-UNDERLAY-PEER:
```

Comment:
@ClausHolbechArista - we are seeing this extra diff because of #2569 - let's merge that PR first then rebase this one!